src/exec.cpp

```diff
@@ -63,6 +63,9 @@ std::string just_path(const std::string &name)
     if (i == std::string::npos) {
         i = name.rfind('/');
     }
+    if (name.substr(0, i+1).rfind('/')) {
```

Comment:
This doesn't work because the value returned when the character is not found, `npos`, is not zero. Better to rework the whole function to use `find_first_of`.